lib/cisco_node_utils/bgp.rb

```diff
@@ -582,7 +551,7 @@ def default_suppress_fib_pending
     # BGP Timers Getters
     def timer_bgp_keepalive_hold
       match = config_get('bgp', 'timer_bgp_keepalive_hold', @get_args)
-      match.nil? ? default_timer_bgp_keepalive_hold : match.first
+      match.nil? ? default_timer_bgp_keepalive_hold : match
```

Comment:
The config_get token of this property returns two values.  Shouldn't we return match.last?

```
timer_bgp_keepalive_hold:
  config_get_token_append: '/^timers bgp (\d+) (\d+)$/'
  config_set_append: '<state> timers bgp <keepalive> <hold>'
```